packages/models-library/src/models_library/api_schemas_webserver/_base.py

```diff
@@ -70,3 +78,12 @@ def data_json(
             exclude_none=exclude_none,
             **kwargs,
         )
+
+
+#
+# mapping tools
+#
+
+
+def copy_dict(data: dict, update_keys: dict[str, str]) -> dict[str, Any]:
```

Comment:
I am very confused about the naming here. Please rename it and and a docstring.

You are generating a new dictionary where you put the values form `update_keys` dict inside the keys of `data` dic.
You are just mapping values to values not copying a dict.

Reply:
The pervious name at least provided more context to what this was doing. I prefer the previous naming, since I find it clear.

Reply:
Let me explain my thinking here.
I was trying to replicate an interface we are familiar with. Specifically https://docs.pydantic.dev/1.10/usage/exporting_models/#modelcopy 
```python

model.copy(update={"current_value":"new_value"})

```
where you return a copy of `model` with updated values. IN my case, I wanted updated keys so I did call `update_keys`.